src/axolotl/utils/logging.py

```diff
@@ -0,0 +1,18 @@
+import logging
+
+from axolotl.utils.distributed import is_main_process
+
+
+def log_info_rank_zero(log: logging.Logger, message: str):
```

Comment:
FYI, we'll need to pass the `use_environ` arg when appropriate (pre-deepspeed init)